README.md

```diff
@@ -14,7 +14,7 @@ The Go types and validation should be compatible with the current Go release; ea
 
 Additional documentation about how this group operates:
 
-- [Code of Conduct](https://github.com/opencontainers/tob/blob/d2f9d68c1332870e40693fe077d311e0742bc73d/code-of-conduct.md)
+- [Code of Conduct](code-of-conduct)
```

Comment:
Ah, I missed that this isn't actually a correct URL -- it should be `[code-of-conduct]`.

Reply:
Ack, my mistake I'll fix this.